linetools/spectra/xspectrum1d.py

```diff
@@ -468,6 +497,10 @@ def plot(self, **kwargs):
 
         ax.set_xlim(xmin, xmax)
 
+        # Labels
+        ax.set_xlabel('Wavelength', size=16)
```

Comment:
I've done so for wavelength.  Am avoiding it for Flux for
now as it is a bit of a pain when the unit is dimensionless.